databricks/koalas/series.py

```diff
@@ -319,6 +319,64 @@ def unique(self):
 
     @derived_from(pd.Series)
     def value_counts(self, normalize=False, sort=True, ascending=False, bins=None, dropna=True):
+        """
+        Return a Series containing counts of unique values.
+        The resulting object will be in descending order so that the
+        first element is the most frequently-occurring element.
+        Excludes NA values by default.
+        Parameters
+        ----------
+        normalize : boolean, default False
+            If True then the object returned will contain the relative
+            frequencies of the unique values.
+        sort : boolean, default True
+            Sort by values.
+        ascending : boolean, default False
+            Sort in ascending order.
+        bins : Not Yet Supported
+        # TODO: Update Documentation when its supported
+        dropna : boolean, default True
+            Don't include counts of NaN.
+        Returns
+        -------
+        counts : Series
+        See Also
+        --------
+        Series.count: Number of non-NA elements in a Series.
+        Examples
+        --------
+        import pandas as pd
+        import databricks.koalas as ks
+        import numpy as np
```

Comment:
We don't need to import these here since they are already imported at the top of this file.